cmds/core/ip/ip_linux.go

```diff
@@ -148,15 +148,15 @@ func parseFlags(args []string, out io.Writer) (cmd, error) {
 
 	fs.Usage = func() {
 		fmt.Fprintf(out, "%s\n\n", ipHelp)
-
 		fs.PrintDefaults()
 	}
 
-	fs.Parse(unixflag.ArgsToGoArgs(args[1:]))
-	cmd.Args = fs.Args()
+	if err := fs.Parse(unixflag.ArgsToGoArgs(args[1:])); err != nil {
+		return cmd, fmt.Errorf("error parsing flags: %w", err)
```

Comment:
I'm more and more uncomfortable with the repetition  of 'error parsing flags' text. It's an informal standard that people are going to fail to follow over time. And, again, are you sure that fs.Parse will return on error? 
Looking at the above, it uses: flag.ExitOnError)
so the effect of this change is to add code we don't need, code that in fact will never be executed (which makes coverage stats WORSE), because golangci-lint has a superficial understanding of what's really going on.